verifiers/internal/gha/provenance.go

```diff
@@ -286,7 +309,8 @@ func isValidDelegatorBuilderID(prov iface.Provenance) error {
 }
 
 // VerifyProvenance verifies the provenance for the given DSSE envelope.
-func VerifyProvenance(env *dsselib.Envelope, provenanceOpts *options.ProvenanceOpts, trustedBuilderID *utils.TrustedBuilderID, byob bool) error {
+func VerifyProvenance(env *dsselib.Envelope, provenanceOpts *options.ProvenanceOpts, trustedBuilderID *utils.TrustedBuilderID, byob bool,
+	builderOpts *options.BuilderOpts) error {
```

Comment:
nit: Since we only use `builderOpts.ExpectedID`, would it make sense to just pass the expected ID here rather than the entire `BuilderOpts`?

Reply:
Yes good catch, thanks!